torch/_torch_docs.py

```diff
@@ -7296,11 +7296,9 @@ def merge_dicts(*dicts):
 The :attr:`std` is a tensor with the standard deviation of
 each output element's normal distribution
 
-The shapes of :attr:`mean` and :attr:`std` don't need to match, but the
-total number of elements in each tensor need to be the same.
-
-.. note:: When the shapes do not match, the shape of :attr:`mean`
-          is used as the shape for the returned output tensor
+The shapes of :attr:`mean`, :attr:`std`, and :attr:`out` (if present) must
+match exactly.  Unless the size of :attr:`out` is zero, then it will be resized
+based on the supplied arguments, which must still match.
```

Comment:
This is probably not correct. Could it be the case that
```suggestion
The shapes of :attr:`mean` and :attr:`std` must be broadcastable. 
```
given that you're using `infer_size` in the previous PR.

I would not mention the `out=` argument here, as the behaviour of that argument is the same all across the PyTorch library.

Reply:
I see in the example in the OP that the following now breaks:
```python
torch.normal(torch.rand(3, 4, 5), torch.rand(3, 4, 1))
```
This does not seem right, as their shapes are broadcastable. We should support this behaviour. In fact, I believe the only change needed to support this behaviour is to remove the `CHECK_NORMAL_SIZES` check. Note that the fact that `mean` and `std` are broadcastable is checked within `at::infer_size` in the meta function, so we don't need to add any additional checks for that.

Reply:
i'll look into it and follow up later. wasn't sure what behavior is desirable here